assets/js/blocks/product-results-count/edit.tsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import { useBlockProps } from '@wordpress/block-editor';
+import { Disabled } from '@wordpress/components';
+import type { BlockEditProps } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import Block from './block';
+
+export interface Attributes {
+	className?: string;
+}
+
+const Edit = ( { attributes }: BlockEditProps< Attributes > ) => {
+	const { className } = attributes;
+	const blockProps = useBlockProps( {
+		className: classNames( 'wc-block-product-results-count', className ),
+	} );
+
+	return (
+		<>
+			<div { ...blockProps }>
+				<Disabled>
```

Comment:
Similar to [a comment I left in #8157](https://github.com/woocommerce/woocommerce-blocks/pull/8157#discussion_r1071025953), I think we can remove the `Disabled` component if there are no links/buttons inside the block.